bundles/org.openhab.binding.automower/src/main/resources/OH-INF/update/update.xml

```diff
@@ -0,0 +1,288 @@
+			</add-channel>
+			<add-channel id="number-of-charging-cycles" groupIds="statistics">
+				<type>automower:statNumberOfChargingCyclesType</type>
+			</add-channel>
+			<add-channel id="number-of-collisions" groupIds="statistics">
+				<type>automower:statNumberOfCollisionsType</type>
+			</add-channel>
+			<add-channel id="total-charging-time" groupIds="statistics">
+				<type>automower:statTotalChargingTimeType</type>
+			</add-channel>
+			<add-channel id="total-cutting-time" groupIds="statistics">
+				<type>automower:statTotalCuttingTimeType</type>
+			</add-channel>
+			<add-channel id="total-cutting-percent" groupIds="statistics">
+				<type>automower:statTotalCuttingPercentType</type>
+			</add-channel>
+			<add-channel id="total-drive-distance" groupIds="statistics">
+				<type>automower:statTotalDriveDistanceType</type>
+			</add-channel>
+			<add-channel id="total-running-time" groupIds="statistics">
+				<type>automower:statTotalRunningTimeType</type>
+			</add-channel>
+			<add-channel id="total-searching-time" groupIds="statistics">
+				<type>automower:statTotalSearchingTimeType</type>
+			</add-channel>
+			<add-channel id="total-searching-percent" groupIds="statistics">
+				<type>automower:statTotalSearchingPercentType</type>
+			</add-channel>
+			<add-channel id="up-time" groupIds="statistics">
+				<type>automower:statUpTimeType</type>
+			</add-channel>
+
+			<!-- Calendar Tasks -->
+			<remove-channel id="calendar-tasks"/>
+
+			<!-- optional channels
+				<add-channel id="<xx>-start" groupIds="calendartasks">
+				<type>automower:calendarTasksStartType</type>
+				</add-channel>
+				<add-channel id="<xx>-duration" groupIds="calendartasks">
+				<type>automower:calendarTasksDurationType</type>
+				</add-channel>
+				<add-channel id="<xx>-monday" groupIds="calendartasks">
+				<type>automower:calendarTasksMondayType</type>
+				</add-channel>
+				<add-channel id="<xx>-tuesday" groupIds="calendartasks">
+				<type>automower:calendarTasksTuesdayType</type>
+				</add-channel>
+				<add-channel id="<xx>-wednesday" groupIds="calendartasks">
+				<type>automower:calendarTasksWednesdayType</type>
+				</add-channel>
+				<add-channel id="<xx>-thursday" groupIds="calendartasks">
+				<type>automower:calendarTasksThursdayType</type>
+				</add-channel>
+				<add-channel id="<xx>-friday" groupIds="calendartasks">
+				<type>automower:calendarTasksFridayType</type>
+				</add-channel>
+				<add-channel id="<xx>-saturday" groupIds="calendartasks">
+				<type>automower:calendarTasksSaturdayType</type>
+				</add-channel>
+				<add-channel id="<xx>-sunday" groupIds="calendartasks">
+				<type>automower:calendarTasksSundayType</type>
+				</add-channel>
+				<add-channel id="<xx>-workAreaId" groupIds="calendartasks">
+				<type>automower:calendarTasksWorkAreaIdType</type>
+				</add-channel>
+				<add-channel id="<xx>-workArea" groupIds="calendartasks">
+				<type>automower:workareaNameType</type>
+				</add-channel>
+			-->
```

Comment:
Use er remove.

Reply:
removed channels that are dynamically handled by the binding and therefore not specified in the update instruction.